### PR TITLE
refactor(app): replace ButterKnife with View Binding

### PR DESCRIPTION
**Pull request by rosariopfernandes**
This should replace the deprecated ButterKnife library with View Binding in our sample app.

### CODE REVIEW

File: auth/src/main/java/com/firebase/ui/auth/viewmodel/AuthViewModelBase.java [hunk 1]
```diff
@@ -2,7 +2,6 @@
 
 import android.app.Application;
 
-import com.firebase.ui.auth.AuthUI;
```
Comment (rosariopfernandes) [inline auth/src/main/java/com/firebase/ui/auth/viewmodel/AuthViewModelBase.java:5]: This change is unrelated to this PR.
I'm not sure how it ended up here but it is harmless, so I guess it's ok to keep it?

Comment (samtstern) [reply auth/src/main/java/com/firebase/ui/auth/viewmodel/AuthViewModelBase.java:5]: Yep not a problem.

**Review by samtstern — APPROVED** [review]

**Comment by samtstern** [comment]
🙏  thank you so much for this @rosariopfernandes!  Goodbye ButterKnife 👋
